voc/sample.js

```diff
@@ -0,0 +1,45 @@
+var module = angular.module('uploader', []);
+/**
+ * XMLHttpRequest wrapper that supports file upload progress since $http doesn't
+ *
+ * Usage similar to $http, returns a promise from the send method
+ */
+
+module.service('uploader', ['$q', function($q) {
+  function readyStateChange(deferred, xhr) {
+    if(xhr.readyState == 4) {
```

Comment:
At column 5: [(indent) Expected indentation of 6 space characters but found 4.](http://.../indent)